packages/core/index.d.ts

```diff
@@ -30,7 +30,7 @@ export interface Request<
   event: TEvent
   context: TContext
   response: TResult | null
-  earlyResponse: TResult | null
+  earlyResponse?: TResult | null
```

Comment:
Let's add `undefined` as a possible value too, based on https://github.com/middyjs/middy/issues/1275#issuecomment-2687934428

Reply:
Sounds good

Reply:
So, is `null` a valid value? At least it works during runtime and matches the `response` type that way.

Reply:
Yes, `null` is a valid response